template-manager-core/src/main/java/com/sequenceiq/cloudbreak/template/views/RdsView.java

```diff
@@ -59,6 +58,12 @@
     private final RdsViewDialect rdsViewDialect;
 
     public RdsView(RDSConfig rdsConfig) {
+        this(rdsConfig, "");
+    }
+
+    public RdsView(RDSConfig rdsConfig, String sslCertificateFile) {
+        // Note: any value is valid for sslCertificateFile for sake of backward compatibility.
+        this.sslCertificateFile = Objects.requireNonNullElse(sslCertificateFile, "");
```

Comment:
The same here, it is not the file, it's just the location/path of the file.

Reply:
Good point. Fixed.